src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/SearchBox/BitSearchBoxDemo.razor.cs

```diff
@@ -155,5 +165,29 @@ public partial class BitSearchBoxDemo
         private readonly string example4HTMLCode = @"<BitSearchBox Placeholder=""Search"" Width=""250px""></BitSearchBox>";
 
         private readonly string example5HTMLCode = @"<BitSearchBox Placeholder=""Search"" IsEnabled=""false""></BitSearchBox>";
+
+        private readonly string example6HTMLCode = @"<EditForm Model=""validationSearchBoxModel"">
+     <DataAnnotationsValidator/>
+     <BitSearchBox DefaultValue = ""This is default value"" @bind-Value=""validationSearchBoxModel.text""/>
+     <ValidationMessage For = ""()=>validationSearchBoxModel.Text"" ></ ValidationMessage >
```

Comment:
Plz remove space between parameter and value and redundant space. for ex: `For = ""()=>validationSearchBoxModel.Text""`